lib/matplotlib/colorbar.py

```diff
@@ -464,12 +463,18 @@ def __init__(self, ax, cmap=None,
             linewidth=mpl.rcParams['axes.linewidth'], closed=True, zorder=2)
         ax.add_artist(self.outline)
         self.outline.set(clip_box=None, clip_path=None)
+
         self.patch = mpatches.Polygon(
             np.empty((0, 2)),
             color=mpl.rcParams['axes.facecolor'], linewidth=0.01, zorder=-1)
         ax.add_artist(self.patch)
 
-        self.dividers = None
+        self.dividers = collections.LineCollection(
+            [],
+            colors=[mpl.rcParams['axes.edgecolor']],
```

Comment:
Doesn't this change the time of `rcParams` resolution from draw-time to init?

Reply:
The same "change in resolution time" occurred in https://github.com/matplotlib/matplotlib/pull/15981, where it is arguably a bugfix.
But sure, added to the changelog.